src/applications/ivc-champva/10-10D/components/Applicant/applicantFileUpload.js

```diff
@@ -115,14 +115,10 @@ export const applicantSchoolCertUploadUiSchema = {
           return (
             <>
               You’ll need to submit a copy of a document showing proof of{' '}
-              <b>{posessive}</b> school enrollment. If{' '}
-              <b>{nonPosessive === 'you' ? 'you’re' : posessive}</b> planning to
-              enroll,{' '}
-              <b>
-                {nonPosessive === 'you' ? 'you’ll' : `${nonPosessive} will`}
-              </b>{' '}
-              need to upload a document showing information about{' '}
-              <b>{posessive}</b> plan to enroll.
+              <b>{posessive}</b> school enrollment. If <b>{posessive}</b>{' '}
+              planning to enroll, <b>{nonPosessive} will</b> need to upload a
+              document showing information about <b>{posessive}</b> plan to
+              enroll.
```

Comment:
All custom `you`/`your` language can be dropped, as we've moved away from directly addressing the user and now just list the applicant's name